Core/SubterfugeCore/Core/Game.cs

```diff
@@ -87,5 +89,41 @@ public void LoadGameEvents(List<GameEventModel> gameEvents)
                 .ConvertAll<GameEvent>(m => GameEventFactory.parseGameEvent(m))
```

Comment:
This is where we load the GameEvents from the server. Could you also add a case for the DRILL_MINE event type so that we can convert these events from the server in the `GameEventFactory`? See [this file](https://github.com/Subterfuge-Revived/Remake-Core/blob/981b97743aa8ee4a650ca9ac32b3235a44a80bb5/Core/SubterfugeCore/Core/GameEvents/Base/GameEventFactory.cs#L6)

Should be a pretty simple addition.